sdk/search/search-documents/src/serviceUtils.ts

```diff
@@ -331,11 +370,13 @@ export function convertTokenizersToPublic(
   const result: LexicalTokenizer[] = [];
   for (const tokenizer of tokenizers) {
     if (tokenizer.odatatype === "#Microsoft.Azure.Search.PatternTokenizer") {
+      const patternTokenizer = tokenizer as PatternTokenizer;
+      const flags: RegexFlags[] | undefined = patternTokenizer.flags
```

Comment:
why is the type set explicitly, wouldn't it be inferred?

Reply:
It doesn't need to be, and it also doesn't need to be a ternary. I went ahead and removed that, thanks!